cargo-afl/tests/integration.rs

```diff
@@ -46,3 +46,46 @@ fn integration() {
     }
     assert!(temp_dir_path.join("default").join("fuzzer_stats").is_file());
 }
+
+#[test]
+fn integration_cfg() {
+    let temp_dir = tempfile::TempDir::new().expect("Could not create temporary directory");
+    let temp_dir_path = temp_dir.path();
+
+    assert_cmd::Command::new(cargo_afl_path())
+        .arg("afl")
+        .arg("build")
+        .arg("--example")
+        .arg("cfg")
+        .env("AFL_NO_CFG_FUZZING", "1")
+        .arg("-vv")
+        .arg("--manifest-path")
+        .arg("../afl/Cargo.toml")
+        .assert()
+        .success();
+
+    let mut child = process::Command::new(cargo_afl_path())
+        .arg("afl")
+        .arg("fuzz")
+        .stdout(process::Stdio::inherit())
+        .stderr(process::Stdio::inherit())
+        .arg("-i")
+        .arg(input_path())
+        .arg("-o")
+        .arg(temp_dir_path)
+        .arg(examples_path("cfg"))
+        .env("AFL_NO_UI", "1")
+        .spawn()
+        .expect("Could not run cargo afl fuzz");
+    thread::sleep(time::Duration::from_secs(5));
+    for _ in 0..5 {
+        thread::sleep(time::Duration::from_secs(1));
+        child.kill().unwrap_or_default();
+    }
+    assert!(temp_dir_path.join("default").join("fuzzer_stats").is_file());
+    let crashes = std::fs::read_dir(temp_dir_path.join("default").join("crashes"))
+        .unwrap()
+        .count()
+        - 1;
```

Comment:
The `- 1` is for README.txt, right?

I suspect this will be confusing for many. Could we please remove the `- 1` and make the check `crashes >= 1`?

Reply:
Yes. It's done!

Reply:
There is also an env that prevents creating the readme

Reply:
Oh, [I see it now](https://aflplus.plus/docs/env_variables/): `AFL_NO_CRASH_README`.

That's very useful. Thanks you!